scripts/casestudies/casestudy_template.yml

```diff
@@ -0,0 +1,64 @@
+id: #Unique id that will be used on website. So in case of id that is "test" people will navigate to /casestudies/test to read about "test" case study.
+company: #We need basic information about a company so people know the case study is real and creditable.
+  name: #Company name.
+  description: #Shor description of most important things people should know about the company.
+  customers: #Amount of customers to indicate the impact that the company has.
+  industry: #Is it retail? software development? put the same value that company has on their LinkedIn page.
+  revenue: #Specify company earnings to indicate to case study reader how critical online operations are.
+  website: #Link to main website.
+  logo: #Link to logo of the company in SVG format.
+  contact:
+  #Let us know who we can contact you, some generic email or just a name and link to some social media. We need this to be able to:
+  #- validate long term if case study is still valid
+  #- ping you when we know we plan to change some parts of AsyncAPI that you are already using
+  - name: #Full name of the contact person.
+    link: #Specify where people can find contact person.
+challenges: |
+    #Describe want challanges company have, that faced them towards AsyncAPI. There is no limit here. Use [CommonMark](https://commonmark.org/).
+solution: |
+    #Describe the solution for the challange and what role AsyncAPI played in it. There is no limit here. Use [CommonMark](https://commonmark.org/)
+technical: #We need some more technical information related to case study.
+  languages:
+    - #Provide a list of programming lanugages people using AsyncAPI work with.
+  frameworks:
+    - #Specify what frameworks they use, maybe Spring in Java or Nest.js in JavaScript.
+  protocols:
+    - #What protocols are used by the company in relation to AsyncAPI.
+  brokers: |
+    #Provide some details about the broker that is used at the company and what is the setup, size, like of example cluster size, number of topics (in case of Kafka for example).
+  testing: |
+    #Explain how testing of the solution is done. How you ensure API works as expected after changes, that users are not affected. For example how do you know who will be affected if you stop sending some message? Provide examples if possible.
+  architecture: |
+    #Explain the setup of the EDA architecture. Perfect woudl be if you could point what [enterprise integration patterns](https://www.enterpriseintegrationpatterns.com/patterns/messaging) you applied. Provide examples if possible.
```

Comment:
```suggestion
    #Explain the setup of the EDA architecture. It would be perfect if you could point out what [enterprise integration patterns](https://www.enterpriseintegrationpatterns.com/patterns/messaging) you applied. Provide examples if possible.
```